terra-jupyter-r/scripts/kernelspec.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+TMP_KERNELSPEC_DIR=$1
+KERNELSPEC_HOME=$2
+
+# Replace the contents of the Python kernel scripts
+sed -e 's/${PY_VERSION}/3/g' -e 's|${JUPYTER_HOME}|'${JUPYTER_HOME}'|g' ${TMP_KERNELSPEC_DIR}/python_kernelspec.tmpl > ${KERNELSPEC_HOME}/python3/kernel.json
```

Comment:
is this file committed accidentally or you intend to enable python(the path is wrong if you do, there's already `terra-jupyter-r/scripts/kernel/kernelspec.sh`)?

Reply:
This is accidental. I'll reset.